docs/quickstarts/excel-quickstart-vue.md

```diff
@@ -42,176 +38,170 @@ Each add-in requires a manifest file to define its settings and capabilities.
+       padding: 15px;
+     }
+   </style>
+   ```
 
 ## Start the dev server
 
 1. From the terminal, run the following command to start the dev server.
 
-    ```command&nbsp;line
-    npm start
-    ```
+   ```command line
+   npm run serve
+   ```
 
-2. In a web browser, navigate to `https://localhost:8080`. If your browser indicates that the site's certificate is not trusted, you will need to configure your computer to trust the certificate. 
+2. In a web browser, navigate to `https://localhost:3000` (notice the `https`). If your browser indicates that the site's certificate is not trusted, you will need to configure your computer to trust the certificate.
 
-3. After your browser loads the add-in page without any certificate errors, you're ready test your add-in.
+3. When the page on `https://localhost:3000` is blank and without any certificate errors, it means that it is working. We are mounting the Vue App only after Office is initialized, so it will only show something inside an Excel environment.
 
 ## Try it out
 
 1. Follow the instructions for the platform you'll be using to run your add-in and sideload the add-in within Excel.
 
-    - Windows: [Sideload Office Add-ins on Windows](../testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md)
-    - Web browser: [Sideload Office Add-ins in Office on the web](../testing/sideload-office-add-ins-for-testing.md#sideload-an-office-add-in-in-office-on-the-web)
-    - iPad and Mac: [Sideload Office Add-ins on iPad and Mac](../testing/sideload-an-office-add-in-on-ipad-and-mac.md)
+   - Windows: [Sideload Office Add-ins on Windows](../testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md)
+   - Web browser: [Sideload Office Add-ins in Office on the web](../testing/sideload-office-add-ins-for-testing.md#sideload-an-office-add-in-in-office-on-the-web)
+   - iPad and Mac: [Sideload Office Add-ins on iPad and Mac](../testing/sideload-an-office-add-in-on-ipad-and-mac.md)
 
 2. In Excel, choose the **Home** tab, and then choose the **Show Taskpane** button in the ribbon to open the add-in task pane.
 
-    ![Excel add-in button](../images/excel-quickstart-addin-2a.png)
+   ![Excel add-in button](../images/excel-quickstart-addin-2a.png)
 
 3. Select any range of cells in the worksheet.
 
 4. In the task pane, choose the **Set color** button to set the color of the selected range to green.
 
-    ![Excel add-in](../images/excel-quickstart-addin-2c.png)
+   ![Excel add-in](../images/excel-quickstart-addin-2c.png)
 
 ## Next steps
 
 Congratulations, you've successfully created an Excel task pane add-in using Vue! Next, learn more about the capabilities of an Excel add-in and build a more complex add-in by following along with the Excel add-in tutorial.
 
-> [!div class="nextstepaction"]
-> [Excel add-in tutorial](../tutorials/excel-tutorial.md)
+> [!div class="nextstepaction"][excel add-in tutorial](../tutorials/excel-tutorial.md)
```

Comment:
I think this change may be the cause of the warning. Can you try reverting this and see if that helps?

Note that I'm not going to be the person to sign off.

Reply:
thanks, that fixed it.